webapp/src/components/PayMortgagePage/PayMortgagePage.js

```diff
@@ -98,7 +101,7 @@ export default class PayMortgagePage extends React.PureComponent {
                   </Link>
                 ),
                 outstanding_amount: formatMana(
-                  mortgage ? mortgage.outstanding_amount : 0
+                  mortgage ? getMortgageOutstandingAmount(mortgage) : 0
```

Comment:
maybe unnecessary if we decide to do the check on `getMortgageOutstandingAmount`